README.md

```diff
@@ -493,8 +489,6 @@ Services to securely store your Docker images.
 * [Container solutions](http://container-solutions.com/blog/)
 * [Container42](http://container42.com/)
 * [Docker Cheat Sheet](http://docker.jens-piegsa.com) by [@JensPiegsa][JensPiegsa] *(updated for Docker 1.13)*
-* [Docker Cheat Sheet][docker-cheat-sheet] by [@wsargent][wsargent] __MUST SEE__
```

Comment:
I think you're right. First, I wanted to list it under 'Useful Resources'. I read through the cheat sheet and it looks good for both beginners and experienced users. In addition, as you stated it's well maintained.